### DISCUSSION TITLE
NRF24l01 how to sent byte array

### DISCUSSION THREAD

**Discussion by derungebundener** [jgromes/RadioLib#363] [Q&A]
A silly question - we can recieve byte array, but in an example i cant find how to send it. Can any one help please? Thanks.

**Comment by jgromes** [comment]
That is indeed missing in the examples, however, it's exactly the same as for any other module, so just

```c++
byte array[8] = { 0, 1, 2, 3, 4, 5, 6, 7 };
radio.transmit(array, 8);
```

should work.

**Reply by derungebundener** [reply]
Thanks so much for the answer :)
